draft-ietf-quic-transport.md

```diff
@@ -3471,6 +3471,10 @@ Datagrams containing Initial packets MAY exceed 1200 bytes if the client
 believes that the Path Maximum Transmission Unit (PMTU) supports the size that
 it chooses.
 
+UDP datagrams MUST NOT be fragmented at the IP layer.  In IPv4
+{{!IPv4=RFC0791}}, the DF bit MUST be set to prevent fragmentation on the path.
+Endpoints SHOULD discard fragmented IP packets.
```

Comment:
I believe this is desirable at the receiving endpoint, but saying  "Endpoints SHOULD discard fragmented IP packets." ... isn't something that the transport has control over. I suggest you remove this. It seems sufficient to say at the sender don't do IPv4 or IPv6 fragmentation and do set FD